src/main/java/session/MenuSession.java

```diff
@@ -0,0 +1,224 @@
+package session;
+
+import auth.BasicAuth;
+import auth.HqAuth;
+import beans.NewSessionResponse;
+import hq.CaseAPIs;
+import install.FormplayerConfigEngine;
+import objects.SerializableMenuSession;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.io.output.ByteArrayOutputStream;
+import org.commcare.api.persistence.UserSqlSandbox;
+import org.commcare.api.session.SessionWrapper;
+import org.commcare.session.SessionFrame;
+import org.commcare.suite.model.SessionDatum;
+import org.commcare.util.cli.*;
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.util.OrderedHashtable;
+import org.javarosa.core.util.externalizable.DeserializationException;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+import org.javarosa.xpath.XPathException;
+import org.javarosa.xpath.XPathParseTool;
+import org.javarosa.xpath.expr.XPathExpression;
+import org.javarosa.xpath.expr.XPathFuncExpr;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import repo.SessionRepo;
+import services.InstallService;
+import services.RestoreService;
+import util.StringUtils;
+
+import java.io.ByteArrayInputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.UUID;
+
+/**
+ * Created by willpride on 2/5/16.
+ */
+@Component
+public class MenuSession {
+    FormplayerConfigEngine engine;
```

Comment:
i'm not running this through the linter, but assuming that the ones that are marked non private are meant to be non-private